test/test_rocrand_generate.cpp

```diff
@@ -145,10 +156,9 @@ TEST_P(rocrand_generate_tests, short_test)
 
 TEST(rocrand_generate_tests, neg_test)
```

Comment:
After thinking on this a bit longer, why was it chosen to add the null check in the existing tests and not extend the "null" test into a [typed test suite](http://google.github.io/googletest/reference/testing.html#TYPED_TEST_SUITE)?

Reply:
@rponnuru5 would you be able to refactor your changes to meet Nara's suggestion?

Reply:
@Naraenda , Pls provide more details about it.

Reply:
I meant grouping the `nullptr` tests together with the already existing `nullptr` tests. Having the `nullptr` test in the generator parameterized tests doesn't make sense since we don't instantiate the different types of generators. There's no point in testing the same things multiple times with the exact same input parameters. I've implemented it here: https://github.com/ROCm/rocRAND/pull/519